app/views/admin/creator_settings/new.html.erb

```diff
@@ -3,7 +3,7 @@
     <%= Rails.application.assets["setup-mode.css"].to_s.html_safe %>
   </style>
 
-  <main id="main-content" class="flex flex-1 justify-center flex-col crayons-layout crayons-layout--limited-xs" data-controller="creator-settings">
+  <main id="main-content" class="flex flex-1 justify-center flex-col crayons-layout crayons-layout--limited-s" data-controller="creator-settings">
```

Comment:
I changed this since we have longer labels and it looked more spacious design wise, but we are planning on getting a final design thumbs up before we release.